core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -0,0 +1,410 @@
+        return send(arguments, null, new PollingTransactionReceiptProcessor(caver, 1000, 15));
+    }
+
+    /**
+     * Send a transaction to smart contract and execute its method.
+     * It sets TransactionReceiptProcessor to PollingTransactionReceiptProcessor.
+     * @param arguments A List of parameter that solidity wrapper type to call smart contract method.
+     * @param options An option to execute smart contract method.
+     * @return TransactionReceiptData
+     * @throws IOException
+     * @throws TransactionException
+     * @throws ClassNotFoundException
+     * @throws NoSuchMethodException
+     * @throws InvocationTargetException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     */
+    public TransactionReceipt.TransactionReceiptData send(List<Object> arguments, SendOptions options) throws IOException, TransactionException, ClassNotFoundException, NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
+        return send(arguments, options, new PollingTransactionReceiptProcessor(caver, 1000, 15));
+    }
+
+    /**
+     * Send a transaction to smart contract and execute its method.
+     * @param arguments A List of parameter that solidity wrapper type to call smart contract method.
+     * @param options An option to execute smart contract method.
+     * @param processor A TransactionReceiptProcessor to get receipt.
+     * @return TransactionReceiptData
+     * @throws IOException
+     * @throws TransactionException
+     * @throws ClassNotFoundException
+     * @throws NoSuchMethodException
+     * @throws InvocationTargetException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     */
+    public TransactionReceipt.TransactionReceiptData send(List<Object> arguments, SendOptions options, TransactionReceiptProcessor processor) throws IOException, TransactionException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+        List<Object> functionParams = new ArrayList<>();
+
+        if(arguments != null) {
+            functionParams.addAll(arguments);
+        }
+
+        if(options == null) {
+            if(getDefaultSendOptions() == null) {
+                throw new NullPointerException("Default SendOptions instance is not existed. Please set default SendOptions instance through contract's setDefaultSendOptions.");
+            }
+            options = getDefaultSendOptions();
+        }
+
+
+        checkTypeValid(functionParams);
+
+        String encodedFunction = ABI.encodeFunctionCall(this, arguments);
+
+        SmartContractExecution smartContractExecution = new SmartContractExecution.Builder()
+                .setKlaytnCall(caver.rpc.klay)
+                .setFrom(options.getFrom())
```

Comment:
Can you check `from` and `gas` are not null?
And the user can optionally define values(from example, {from: '0x..'} or {gas: '0x..'} if they want to overwrite what they defined for default sendOptions values) when passing the sendOptions parameter.